man/avocado.rst

```diff
@@ -610,17 +639,16 @@ Here is how to run the sleeptest example test in a remote machine with IP
 address 192.168.0.123 (standard port 22), remote user name `fedora` and
 remote user password `123456`::
 
- $ avocado run --remote-hostname 192.168.0.123 --remote-username fedora --remote-password 123456
+    $ avocado run sleeptest.py --remote-hostname 192.168.0.123 --remote-username fedora --remote-password 123456
 
 The output should look like::
 
- REMOTE LOGIN  : fedora@192.168.0.123:22
- JOB ID    : <JOBID>
- JOB LOG   : /home/<user>/avocado/job-results/job-<date>-<shortid>/job.log
- TESTS     : 1
- (1/1) sleeptest.py:  PASS (1.01 s)
- RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0
- TESTS TIME : 1.01 s
+    JOB ID     : 4733dc1638b78c9af90187ca91f547058edf8c14
+    JOB LOG    : /home/<user>/avocado/job-results/job-2017-03-03T10.33-4733dc1/job.log
+    LOGIN      : fedora@localhost:22 (TIMEOUT: 60 seconds)
+     (1/1) sleeptest.py:SleepTest.test: PASS (1.02 s)
+    RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0
```

Comment:
Also above we have section `FILES`, how about list `/etc/avocado/conf.d` as well?